columnflow/selection/example.py

```diff
@@ -7,8 +7,6 @@
 from collections import defaultdict
 
 from columnflow.selection import Selector, SelectionResult, selector
-from columnflow.production.mc_weight import mc_weight
-from columnflow.production.seeds import deterministic_seeds
```

Comment:
As stated in my other comment, I don't really see a reason to remove these modules from a general example. But maybe I'm missing something

Reply:
See above :)